.circleci/config.yml

```diff
@@ -41,8 +41,8 @@ default_job: &default_job
   steps:
     - shared_steps
     # Run the tests against multiple versions of Rails
-    - run: bundle exec appraisal install
-    - run: bundle exec appraisal rspec
+    # - run: bundle exec appraisal install
+    - run: bundle exec rspec
```

Comment:
reasonable to comment out `appraisal` for now, but i don't think we need this line. it's already a part of `shared_steps`

Reply:
i think instead just comment out the two `appraisal` lines?
also add a comment if you don't mind?

Reply:
sg, updated that with a more descriptive comment 👍